pkg/domain/infra/abi/system.go

```diff
@@ -198,23 +198,24 @@ func (ic *ContainerEngine) SystemDf(ctx context.Context, options entities.System
 		iid, _ := c.Image()
 		state, err := c.State()
 		if err != nil {
+			if errors.Is(err, define.ErrNoSuchCtr) {
+				continue
+			}
 			return nil, fmt.Errorf("failed to get state of container %s: %w", c.ID(), err)
 		}
 		conSize, err := c.RootFsSize()
 		if err != nil {
-			if errors.Is(err, storage.ErrContainerUnknown) {
-				logrus.Error(fmt.Errorf("failed to get root file system size of container %s: %w", c.ID(), err))
-			} else {
-				return nil, fmt.Errorf("failed to get root file system size of container %s: %w", c.ID(), err)
+			if errors.Is(err, storage.ErrContainerUnknown) && errors.Is(err, define.ErrNoSuchCtr) {
```

Comment:
LGTM, and it passes in #23275 and on my laptop ... but shouldn't `&&` be `||`? Line 215 also?

Reply:
good catch, yes of course, will repush tomorrow 

/hold